pkgs/development/python-modules/urwid/default.nix

```
@@ -2,31 +2,13 @@
 
 buildPythonPackage (rec {
```

```suggestion
buildPythonPackage rec {
```

the one at the end of the file as well

---

Done